jumpscale/packages/vdc_dashboard/frontend/components/vdcSettings/Backups.vue

```diff
@@ -152,9 +166,17 @@ module.exports = {
       this.dialogs.actions = true;
     },
     timeDifference(ts) {
-      var timestamp = moment.unix(ts);
-      var now = new Date();
-      return timestamp.to(now);
+      var a = new Date(ts * 1000);
+      var months = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+      var year = a.getFullYear();
+      var month = months[a.getMonth()];
+      var date = a.getDate();
+      var hour = a.getHours();
+      var min = a.getMinutes();
+      var sec = a.getSeconds();
+      var time =
+        date + "/" + month + "/" + year + " " + hour + ":" + min + ":" + sec;
+      return time;
```

Comment:
```suggestion
    return a.toLocaleString();
```

example
![Screenshot from 2021-03-14 16-51-38](https://user-images.githubusercontent.com/10658229/111072975-92311700-84e5-11eb-8ac7-bb2d1b975398.png)